tests/openvino/native/test_nncf_graph_builder.py

```diff
@@ -47,3 +51,20 @@ def test_compare_nncf_graph_omz_models(tmp_path, model_name):
 
     path_to_dot = REFERENCE_GRAPHS_DIR / f"{model_name}.dot"
     compare_nncf_graphs(model, path_to_dot)
+
+
+def test_parallel_edges():
+    model = ParallelEdgesModel().ov_model
+    nncf_graph = GraphConverter.create_nncf_graph(model)
+    input_node = nncf_graph.get_node_by_name("Input")
+    mm_node = nncf_graph.get_node_by_name("Mm")
+    add_node = nncf_graph.get_node_by_name("Add")
+    kwargs = {"dtype": Dtype.FLOAT, "tensor_shape": [1, 3, 3], "parallel_input_port_ids": []}
```

Comment:
Same

Reply:
You mean I should repeat this twice in `NNCFGraphEdge` constructor? And after that to refactor this peace one need to change value in two places instead of one, is it "refactoring friendly"?

Reply:
Or should I create a local variable for each param? What is better?

Reply:
Repeating would be OK here since the entities you are initializing are not related, and have this param set identical only accidentally. Making a local var for this would work too, but wouldn't save anything in terms of line space.

Reply:
Done